pkgs/games/devilutionx/default.nix

```diff
@@ -1,13 +1,13 @@
 { stdenv, fetchFromGitHub, cmake, SDL2, SDL2_mixer, SDL2_ttf, libsodium, pkg-config }:
 stdenv.mkDerivation {
-  version = "unstable-2019-07-28";
+  version = "0.5.0";
   pname = "devilutionx";
 
   src = fetchFromGitHub {
     owner = "diasurgical";
     repo = "devilutionX";
-    rev = "b2f358874705598ec139f290b21e340c73d250f6";
-    sha256 = "0s812km118qq5pzlzvzfndvag0mp6yzvm69ykc97frdiq608zw4f";
+    rev = "88039e9a21f708f07fc0922e5beb96820b5ad607";
```

Comment:
`rev = version;`?

Reply:
I thought that a tag can be moved, a hash stays forever :)
I've put `version`, easier to maintain.